config/segcache.toml

```diff
@@ -6,6 +6,13 @@ host = "0.0.0.0"
 # port listening on
 port = "9999"
 
+# enable the http admin port?
```

Comment:
I considered taking a single `http_listen` parameter and checking for presence or absence to enable/disable the http server, but it's inconsistent with the existing config.

Reply:
I can see us moving to default to http, and allowing old style only with a "legacy" option within a year. You can change how config works when we make that transition.